src/main/resources/application.properties

```diff
@@ -21,4 +21,14 @@ spring.liquibase.change-log=classpath:db/changelog/db.changelog-master.xml
 employee.service.url=http://localhost:8080
 
 #Connect to Benefit application
-benefit.service.url=http://localhost:8082
\ No newline at end of file
+benefit.service.url=http://localhost:8082
+
+#Rabbitmq
+
+# Destination is the name of exchange, group is the name of queue
+# First line for output - exchange name only
+# Second and third line about input - exchange name and queue name
+# Binding name for input consists on method name and operation type (`in`) and index (`0`)
+spring.cloud.stream.bindings.document-out-0.destination=document_exchange
+spring.cloud.stream.bindings.documentInput-in-0.destination=document_exchange
+spring.cloud.stream.bindings.documentInput-in-0.group=documentServiceQueue
```

Comment:
Maybe we don't need to connect to this queue and exchange for inputs since we don't use them